cl/phase1/forkchoice/optimistic/optimistic_impl.go

```diff
@@ -25,13 +25,11 @@ import (
 
 type optimisticStoreImpl struct {
 	opMutex         sync.RWMutex
```

Comment:
Then we can remove this mutex. Thank you : )

Reply:
I would like to that in a follow-up